db/migrate/20161202161114_add_files_collection_ref_to_tribunal_case.rb

```diff
@@ -0,0 +1,5 @@
+class AddFilesCollectionRefToTribunalCase < ActiveRecord::Migration[5.0]
+  def change
+    add_column :tribunal_cases, :files_collection_ref, :uuid, default: 'uuid_generate_v4()', unique: true
```

Comment:
Jesus and I had a chat about this. He pointed out that, if you upload multiple files in one hit, they get uploaded in parallel. So, they would all be the first file in the case, from the uploader's POV, and therefore would all have different collection references. By choosing a reference for the case upfront, we bypass that problem.